spec/server/brisketSpec.js

```diff
@@ -68,6 +68,10 @@ describe("public interface to Brisket", function() {
         expect(Brisket.$).toBe(requireFromLib("application/jquery"));
     });
 
+    it("exposes Brisket.version", function() {
+        expect(Brisket.version).toBeTruthy();
```

Comment:
rather than truthy, you could import package json in the test and verify that window.Brisket matches. that would be a good a way to enforce keeping them in sync